### PR TITLE
Updates import to read with a scanner

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
This PR updates the import reading logic to use a scanner. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-53232

### CODE REVIEW

File: server/app/import.go [hunk 1]
```diff
@@ -20,8 +20,9 @@ import (
 )
 
 const (
-	archiveVersion  = 2
-	legacyFileBegin = "{\"version\":1"
+	archiveVersion    = 2
+	legacyFileBegin   = "{\"version\":1"
+	importMaxFileSize = 1024 * 1024 * 70
```
Comment (sbishel) [inline server/app/import.go:25]: Where is this file size coming from? Should this be from the configuration setting?

Comment (mgdelacroix) [reply server/app/import.go:25]: This comes directly from the Mattermost [slack import service](https://github.com/mattermost/mattermost/blob/master/server/platform/services/slackimport/slackimport.go#L78), but I'm happy to make it configurable

Comment (mgdelacroix) [reply server/app/import.go:25]: After taking a look at how the configuration is generated, making this easy to configure from the plugin's perspective is a bit more involved than expected. We've agreed on reusing MM's approach and we can revisit and expand it later if necessary

**Review by sbishel — APPROVED** [review]
Agree, its not worth the extra work to make it configurable, matching channels value seems reasonable to me.

**Review by esarafianou — APPROVED** [review]

**Pull request merged**